ios/MeasureSDK/MeasureSDK/Config/Config.swift

```diff
@@ -33,5 +35,7 @@ struct Config: InternalConfig, MeasureConfig {
         self.eventsBatchingIntervalMs = 30000 // 30 seconds
         self.maxEventsInBatch = 500
         self.sessionEndThresholdMs = 60000 // 60 seconds
+        self.longPressTimeout = 0.5 // 0.5 seconds
+        self.scaledTouchSlop = 20 //
```

Comment:
The minimum threshold for scroll detection is 3.5 according to flutter's [dragStartDistanceMotionThreshold](https://api.flutter.dev/flutter/widgets/BouncingScrollPhysics/dragStartDistanceMotionThreshold.html) property.